templates/cloud/index.html

```diff
@@ -5,7 +5,9 @@
 {% if meta_description %}{% block meta_description %}{{ meta_description }}{% endblock %}{% endif %}
 
 {% block second_level_nav_items %}
-  {% include "templates/_nav_breadcrumb.html" with section_title="云" page_title="云"  %}
+  {% set section_title="云" %}
```

Comment:
I think it's better to use [`with`](https://jinja.palletsprojects.com/en/2.11.x/templates/#with-statement) so you're making the scope of these variables clear.

Reply:
e.g.

```
{% with section_title="云", page_title="云" %}
  {% include "templates/_nav_breadcrumb.html" %}
{% endwith %}
```

Reply:
all updated